------------------------------------------------------------------------
r878 | bobbybrasko | 2007-05-04 16:59:50 -0400 (Fri, 04 May 2007) | 12 lines

Reported bug fix.

Our automated buildd log filter[1] detected a problem that will cause                                   
your package to segfault on architectures where the size of a pointer                                   
is greater than the size of an integer, such as ia64 and amd64.                                         
                                                                                                        
This is often due to a missing function prototype definition.                                           
                                                                                                        
[1]http://people.debian.org/~dannf/check-implicit-pointer-functions                                     
                                                                                                        
Function `cgdb_malloc' implicitly converted to pointer at kui_driver.c:122

------------------------------------------------------------------------

Index: cgdb/lib/kui/src/kui_driver.c
===================================================================
--- cgdb/lib/kui/src/kui_driver.c	(revision 877)
+++ cgdb/lib/kui/src/kui_driver.c	(revision 878)
@@ -51,6 +51,7 @@
 #include <getopt.h>
 #endif
 
+#include <sys_util.h>
 #include "kui.h"
 #include "kui_term.h"
 
